org/mozilla/jss/util/NativeProxy.java

```diff
@@ -145,28 +181,20 @@ private synchronized static long register() {
      * @param index The index of this object in the registry, as returned
      * from the previous call to register().
```

Comment:
The `@param index` is no longer needed.

Reply:
Since they're private functions I've actually removed both. `HashSet` creation is much simpler than `Hashtable`.